certora/run.js

```diff
@@ -1,37 +1,65 @@
 #!/usr/bin/env node
 
 // USAGE:
-//    node certora/run.js [[CONTRACT_NAME:]SPEC_NAME] [OPTIONS...]
+//    node certora/run.js [[CONTRACT_NAME:]SPEC_NAME]* [--all] [--options OPTIONS...] [--specs PATH]
 // EXAMPLES:
+//    node certora/run.js --all
 //    node certora/run.js AccessControl
 //    node certora/run.js AccessControlHarness:AccessControl
 
-const MAX_PARALLEL = 4;
-
-let specs = require(__dirname + '/specs.json');
-
 const proc = require('child_process');
 const { PassThrough } = require('stream');
 const events = require('events');
-const limit = require('p-limit')(MAX_PARALLEL);
 
-let [, , request = '', ...extraOptions] = process.argv;
-if (request.startsWith('-')) {
-  extraOptions.unshift(request);
-  request = '';
+const argv = require('yargs')
+  // .env('')
+  .options({
+    all: {
+      alias: 'a',
+      type: 'boolean',
+    },
+    spec: {
+      alias: 's',
+      type: 'string',
+      default: __dirname + '/specs.json',
+    },
+    parallel: {
+      alias: 'p',
+      type: 'number',
+      default: 4,
+    },
+    options: {
+      alias: 'o',
+      type: 'array',
+      default: [],
+    },
```

Comment:
This doesn't seem to work as expected. E.g., `-o --optimistic_loop` results in `{ options: [], optimistic_loop: true }`.

We may need a special way to run a single spec with options...

Reply:
I'll check that !

maybe `-o "--optimistic_loop"` is the way to go

Reply:
Note that
```
node ./certora/run.js -o=--optimistic_loop -o=--optimistic_hashing ERC20
```
will not work and that you should do either
```
node ./certora/run.js ERC20 -o=--optimistic_loop -o=--optimistic_hashing
node ./certora/run.js -o=--optimistic_loop -o=--optimistic_hashing -- ERC20
```

Reply:
Ok, I don't think this is a good interface but let's just merge it and see how it works for us.